tests/dat/actions/loggingTimeout.js

```diff
@@ -0,0 +1,35 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more contributor
+// license agreements; and to You under the Apache License, Version 2.0.
+
+// This action prints log lines to stdout for the duration of durationMillis.
+// The output is throttled by a delay of delayMillis between two log lines in
+// order to keep the log small and to stay within the log size limit.
+
+function doLog(startMillis, durationMillis, delayMillis) {
+  var lastLog = 0;
+  var logLines = 0;
+  while (true) {
+     now = new Date();
+     if (now - lastLog > delayMillis) {
+        lastLog = now;
+        logLines++;
+        console.log('[' + logLines + '] The quick brown fox jumps over the lazy dog.');
+     }
+     if (now - startMillis >= durationMillis) {
+        break;
+     }
+  }
```

Comment:
This is a busy loop. On a test system with few CPUs, this can be too heavy weight. Can you please use timeouts or intervals instead?

Reply:
Resolved with the new nodejs:6 action.